core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -158,6 +190,30 @@ export class LoggedInUser extends BaseUser {
     await this.goto(getStartedUrl);
   }
 
+  /**
+   * Function to navigate to the Parents and Teachers page.
+   */
+  async navigateToTeachPage(): Promise<void> {
+    await this.goto(teachUrl);
+    this.page.waitForNavigation();
+  }
+
+  /**
+   * Function to navigate to the Partnerships page.
+   */
+  async navigateToPartnershipsPage(): Promise<void> {
+    await this.goto(partnershipsUrl);
+    this.page.waitForNavigation();
+  }
+
+  /**
+   * Function to navigate to the Volunteer page.
+   */
+  async navigateToVolunteerPage(): Promise<void> {
+    await this.goto(volunteerUrl);
+    this.page.waitForNavigation();
+  }
```

Comment:
Instead of 3 different functions, can we have a common function and pass the URL as a parameter.